tests/conftest.py

```diff
@@ -170,13 +170,24 @@ def _setup_database(_configure_bigchaindb):
 
 @pytest.fixture
 def _bdb(_setup_database, _configure_bigchaindb):
-    yield
     from bigchaindb import config
     from bigchaindb.backend import connect
-    from .utils import flush_db
-    dbname = config['database']['name']
+    from bigchaindb.backend.admin import config as backend_config
+    from bigchaindb.backend.schema import TABLES
+    from .utils import flush_db, update_table_config
     conn = connect()
+    # TODO remove condition once the mongodb implementation is done
+    if config['database']['backend'] == 'rethinkdb':
+        table_configs_before = {
+            t: backend_config(conn, table=t) for t in TABLES
+        }
+    yield
+    dbname = config['database']['name']
     flush_db(conn, dbname)
+    # TODO remove condition once the mongodb implementation is done
+    if config['database']['backend'] == 'rethinkdb':
+        for t, c in table_configs_before.items():
+            update_table_config(conn, t, **c)
```

Comment:
The database config might be nicer in the `_configure_bigchaindb` fixture or a new one.